charts/headlamp/values.schema.json

```
@@ -296,6 +296,22 @@
         "port": {
           "type": "integer",
           "description": "Kubernetes Service port"
+        },
+        "clusterIP": {
+          "type": "string",
+          "description": "Kubernetes Service clusterIP"
+        },
+        "loadBalancerIP": {
+          "type": "string",
+          "description": "Kubernetes Service loadBalancerIP"
+        },
+        "loadBalancerSourceRanges": {
+          "type": "string",
```

This should be array.


```suggestion
"type": "array",
    "items": {
      "type": "string"
}
```